asterius/rts/rts.gc.mjs

```diff
@@ -32,22 +35,72 @@ export class GC {
     for (const p of export_stableptrs) this.stablePtrManager.newStablePtr(p);
     this.symbolTable = symbol_table;
     this.reentrancyGuard = reentrancy_guard;
+    /**
+     * 'Yolo' mode disables garbage collection altogether
+     * (see {@link GC#performGC})
+     * @name GC#yolo
+     */
     this.yolo = yolo;
-    this.gcThreshold = gc_threshold;
+    /**
+     * Garbage collection will not be performed when the
+     * current number of MBlocks is less than {@link GC#gcThreshold}
```

Comment:
Or more precisely, when the current number of "live" MBlocks (as returned by `this.heapAlloc.liveSize()`). "live" means they were allocated by `getMBlocks()` some time ago and hasn't been `freeMBlocks()` yet. Also, this duplicates a bit with the docs at the beginning of `performGC()`, so maybe we should point to that place and improve the description there?

Reply:
Added the precisation. This one comment is still a bit duplicated, but I guess _repetita iuvant_ 😅